lib/src/publication/remote_track_publication.dart

```diff
@@ -40,6 +41,27 @@ class RemoteTrackPublication<T extends RemoteTrack>
   lk_models.VideoQuality _videoQuality = lk_models.VideoQuality.HIGH;
   lk_models.VideoQuality get videoQuality => _videoQuality;
 
+  StreamState _streamState = StreamState.paused;
+
+  /// The server may pause the track when they are bandwidth limitations and resume
+  /// when there is more capacity. This property will be updated when the track is
+  /// paused / resumed by the server. See [TrackStreamStateUpdatedEvent] for the
+  /// relevant event.
+  StreamState get streamState => _streamState;
+
+  @internal
+  Future<void> updateStreamState(StreamState streamState) async {
+    // return if no change
+    if (_streamState == streamState) return;
+    _streamState = streamState;
+    [participant.events, participant.room.events]
```

Comment:
It's firing at Room level here oniichan 😁 @boks1971

Reply:
Got it, thank you thambi (I am going to start calling you thambi which is the opposite of oniichan in my mother tongue Tamil :-) )